src/snowflake/snowpark/_internal/compiler/repeated_subquery_elimination.py

```diff
@@ -178,6 +178,7 @@ def _update_parents(
                 _update_parents(
                     node, should_replace_child=True, new_child=resolved_with_block
                 )
+                self._total_number_ctes += 1
```

Comment:
Hmm. Something is off here; this change is already in main afaict.

Reply:
thanks for spotting, I restored this file to its state from main. I hope this doesn't break the tests.